pymc/distributions/continuous.py

```diff
@@ -673,14 +673,14 @@ class TruncatedNormal(BoundedContinuous):
 
     Parameters
     ----------
-    mu: float
+    mu : tensor_like of float, optional
         Mean.
```

Comment:
This is missing the default, like sigma

Reply:
I was unable to find out the default value for mu and sigma.
could you please guide me, how to identify these dafult value so that it can help me for other methords.

Reply:
should I remove the default values?

Reply:
no, you can review your own PR and ask questions about the parameters or defaults you don't see clear